README.md

```diff
@@ -200,5 +202,9 @@ When a new minor version of Drupal is released:
 1. Test locally with `circleci build --job run-unit-kernel-tests` and so on for
    each job.
 1. Submit a pull request to this repository.
+1. Add a
+   [build configuration](https://hub.docker.com/r/andrewberry/drupal_tests/~/settings/automated-builds/)
+   in Docker Hub with the new tag to insure linked repositories cause a
```

Comment:
Shouldn't be ensure?

https://en.oxforddictionaries.com/usage/ensure-or-insure

Reply:
Apart from this, looks good.

Reply:
Yeeeep.